src/packages/core/components/input-upload-field/input-upload-field.element.ts

```diff
@@ -1,13 +1,15 @@
+import { UmbId } from '../../index.js';
```

Comment:
I think it is better to import from the alias here, no?

```suggestion
import { UmbId } from '@umbraco-cms/backoffice/id';
```